src/StatusTable/__tests__/useStatusTableUtil.test.js

```diff
@@ -0,0 +1,112 @@
+import { filterData, getCompareFormat, mapTableFilters, sortData } from '../useStatusTableUtil';
+
+const FILTER_VALUES = {
+    logId: 'id123',
+    discoveredOn: '2020-01-20'
+};
+
+const DATA = [
+    {
+        log_id: 'tri408/D_20210120_131101/d001/data-01-20-2021_13-11-10.tlog',
```

Comment:
I don't think it's a big deal, but for the sake of public code, let's use test data other than log IDs.

Reply:
Removed all the tri specific data.